client/payment-details/timeline/map-events.js

```diff
@@ -311,6 +311,25 @@ const composeFXString = ( event ) => {
 	);
 };
 
+// Conditionally adds the ARN details to the timeline in case they're available.
+const getRefundTrackingDetails = ( event ) => {
+	return 'available' === event.acquirer_reference_number_status
+		? `Acquirer Reference Number(ARN) ${ event.acquirer_reference_number }`
+		: '';
+};
+
+// Converts the failure reason enums to error messages.
+const getRefundFailureReason = ( event ) => {
+	switch ( event.failure_reason ) {
+		case 'expired_or_canceled_card':
+			return 'the card being expired or canceled.';
```

Comment:
All returned strings here would use `__()` for the localization purpose too.

Reply:
Good catch, added.